### PR TITLE
rgw: improve debugs on the path of RGWRados::cls_bucket_head.

### PR DESCRIPTION
**Pull request by rzarzynski**

### CODE REVIEW

**Comment by rzarzynski** [comment]
jenkins test this please (no log)

**Comment by amitkumar50** [comment]
@rzarzynski Does this PR have no Tracker?

**Comment by rzarzynski** [comment]
@amitkumar50: no, it doesn't. Do we need it? The PR doesn't bring a bug fix nor feature. I would say it's just a clean-up.

File: src/rgw/rgw_common.h [hunk 1]
```diff
@@ -1073,7 +1073,7 @@ struct rgw_bucket {
 WRITE_CLASS_ENCODER(rgw_bucket)
 
 inline ostream& operator<<(ostream& out, const rgw_bucket &b) {
-  out << b.name << "[" << b.marker << "])";
+  out << b.tenant << ":" << b.name << "[" << b.marker << "])";
```
Comment (amitkumar50) [inline src/rgw/rgw_common.h]: @rzarzynski (NIT) Please consider to keep PR title < 50 characters for future PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits

File: src/rgw/rgw_rados.cc [hunk 2]
```diff
@@ -8505,8 +8505,11 @@ int RGWRados::open_bucket_index(const RGWBucketInfo& bucket_info, librados::IoCt
 {
   const rgw_bucket& bucket = bucket_info.bucket;
   int r = open_bucket_index_ctx(bucket_info, index_ctx);
-  if (r < 0)
+  if (r < 0) {
+    ldout(cct, 20) << "open_bucket_index: open_bucket_index_ctx() returned "
```
Comment (amitkumar50) [inline src/rgw/rgw_rados.cc]: @rzarzynski Is there specific purpose of prefixing `open_bucket_index:` in message.
Main_func:called_func sort of?

**Comment by mattbenjamin** [comment]
@rzarzynski I feel bad asking for a rebase;  I can do it, if you don't object

**Comment by rzarzynski** [comment]
@mattbenjamin: no problem at all. :-)

**Comment by cbodley** [comment]
passed qa in http://pulpito.ceph.com/cbodley-2019-05-30_18:03:29-rgw-wip-cbodley-testing-distro-basic-smithi/

**Pull request merged**